LocalExtensions.php

```diff
@@ -5,6 +5,10 @@
 //	require_once( "$IP/extensions/AccessControl/AccessControl.php" );
 //}
 
+if ( $wmgUseAddThis ) {
+	require_once "$IP/extensions/AddThis/AddThis.php";
```

Comment:
consistency is really useful with it enclosed in () as all the ones below.

Reply:
?

Reply:
require_once( "$IP/extensions/AddThis/AddThis.php" );